experiments/srupp_experiments/prepare_corpus.py

```diff
@@ -0,0 +1,14 @@
+
+if __name__ == '__main__':
+    import argparse
```

Comment:
can we move the imports to the top of the file please?

Reply:
will do.